dev-util/radare2/radare2-2.4.0.recipe

```diff
@@ -110,8 +156,10 @@ INSTALL()
 	mv $prefix/data/radare2 $docDir
 	rm -rf $prefix/share
 
-	prepareInstalledDevelLib libr2
+	prepareInstalledDevelLibs libr2 libr_shlr
```

Comment:
Every single `lib:` up there should also be here as well, as they all have an `.so` symlink.